modules/oxide/src/less/theme/components/menu/menu.less

```diff
@@ -35,6 +35,15 @@
     padding: @menu-item-collection-grid-padding;
   }
 
+  @media @breakpoint-lt-sm {
```

Comment:
This needs updating to match the new name done in 517e80ca76eb817cae000d016d64a15544b74711